shell/platform/windows/system_utils_unittests.cc

```diff
@@ -10,22 +10,61 @@
 namespace flutter {
 namespace testing {
 
+class MockWindowsRegistry : public WindowsRegistry {
+ public:
+  virtual ~MockWindowsRegistry() {}
+
+  virtual LSTATUS GetRegistryValue(HKEY hkey,
+                                   LPCWSTR key,
+                                   LPCWSTR value,
+                                   DWORD flags,
+                                   LPDWORD type,
+                                   PVOID data,
+                                   LPDWORD sizeData) const {
+    static const wchar_t* locales =
+        L"en-US\0zh-Hans-CN\0ja\0zh-Hant-TW\0he\0\0";
+    static DWORD locales_len = 35;
+    if (data != nullptr) {
+      if (*sizeData < locales_len) {
+        return ERROR_MORE_DATA;
+      }
+      memcpy(data, locales, locales_len * sizeof(wchar_t));
```

Comment:
Prefix as `std::memcpy` and make sure to `#include <cstring>`, where it's defined.